### ISSUE TITLE
Very low validation accuracy during training

### ISSUE THREAD

**Issue by MaryP111** [ATR-DBI/ScanQA#10]
Hello,

I followed the guidelines for the data processing and the training (run the command: python scripts/train.py --use_color --tag <tag_name>). However, after 40 epochs the top 10 answer accuracy on the validation set is very low (~30%). Could you tell me what might be wrong with the configuration? Thanks in advance! 

![Screenshot from 2022-11-21 00-04-41](https://user-images.githubusercontent.com/22404995/202928762-66d84d34-7a77-401d-aa2c-849cb4ce6dfe.png)
